src/zjs_uart.c

```diff
@@ -161,21 +161,23 @@ static jerry_value_t uart_write(const jerry_value_t function_obj,
 
     zjs_make_promise(promise, NULL, NULL);
 
-    if (!jerry_value_is_object(argv[0])) {
-        DBG_PRINT("first parameter must be a Buffer");
+    uint32_t len;
+    void* data = zjs_serialize_data(argv[0], NULL, &len);
```

Comment:
I am not sure if this should go in. We would need to talk to Sakari because the iotjs spec does not say anything about strings/arrays. I personally think it would be nice for users, but we should talk about it.